bibliopixel/drivers/SPI/errors.py

```diff
@@ -0,0 +1,35 @@
+PERMISSION_ERROR = """Cannot access SPI device.
+Please see
```

Comment:
1. This needs to report the device path.
2. And all that duplication.  Consider extracting it out somehow.